scipy/stats/_covariance.py

```diff
@@ -0,0 +1,174 @@
+from functools import cached_property
+
+import numpy as np
+from scipy import linalg
+
+
+__all__ = ["CovViaPrecision"]
+
+
+class Covariance():
+    """
+    Representation of a covariance matrix
+    """
+
+    def whiten(self, x):
+        """
+        Perform a whitening transformation on data.
+
+        "Whitening" transforms a set of random variables into a new set of
+        random variable with unit-diagonal covariance. When a whitening
```

Comment:
```suggestion
        "Whitening" ("white" as in "white noise", in which each frequency has
        equal power) transforms a set of random variables into a new set of
        random variable with unit-diagonal covariance. When a whitening
```
I remember that the name of another function called `whiten` was questioned recently, and the resolution was to add a statement like this. https://docs.scipy.org/doc/scipy/reference/generated/scipy.cluster.vq.whiten.html

See gh-12446.

We'll see if anyone has another idea for the method name. `decorrelate_and_standardize` is a little long...